### PR TITLE
Fix error with setting repeated proto field for bundling.

### PR DESCRIPTION
**Pull request by bjwatson**
Update bundling tests to use actual proto messages, to prove this fix
works.

Fixes https://github.com/googleapis/gax-python/issues/110

### CODE REVIEW

**Comment by bjwatson** [comment]
@tbetbetbe I know that the pep8 and pylint tests currently fail. They just need to ignore the generated fixture_pb2.py file. Otherwise this is ready to review.

File: tox.ini [hunk 1]
```diff
@@ -6,17 +6,22 @@ envlist = py27,pep8,pylint-errors,pylint-full
...
+           -py.test --timeout=30 --cov-report html --cov-report=term --cov {toxinidir}/google
 
 [testenv:pep8]
 deps = flake8
-commands = flake8 --max-complexity=10 google test --ignore=E501
+commands = flake8 --max-complexity=10 google test --ignore=E501 --exclude='test/*_pb2.py'
 
 [testenv:pylint-errors]
+setenv =
+       PYTHONPATH = {toxinidir}:{toxinidir}/test
```
Comment (geigerj) [inline tox.ini]: Is this because we weren't linting the `test/` directory previously?

Comment (bjwatson) [inline tox.ini]: This wasn't really needed. I got rid of this change.

**Comment by bjwatson** [comment]
Woot! Woot! I finally got a passing build with protoc / Travis CI / tox. :)

**Comment by geigerj** [comment]
LGTM

**Comment by geigerj** [comment]
Note: this adds an implicit dependency of bundling on proto, namely the ClearField method. Ideally this should be moved into gRPC.py and called through the config.py "interface", but that can be done as a follow-up.

**Comment by bjwatson** [comment]
Added TODO comment to code based on feedback

**Pull request merged**